woocommerce/woocommerce
# Cleanup and deprecate task properties and methods

### All Submissions:






### Changes proposed in this Pull Request:



Closes 104-gh-woocommerce/team-ghidorah

Clean up the task list code with the following changes:
- Remove remaining sectioned task list code
- Deprecated unused properties and methods
    - level
    - is_snoozed
    - is_snoozeable
    - get_snoozed_until
    - is_disabled
    - ....
- Remove unused imports
- Update task list type in `@woocommerce/data`


>  One example is content, possibly action button as well - 104-gh-woocommerce/team-ghidorah

We're using `content` and `actionLabel` for [`DefaultTaskItem`](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/tasks/task-list-item.tsx#L149-L179) so I think we should keep them until we update the default task item.






### How to test the changes in this Pull Request:



1.  Go to woocommerce > home
2. Perform a smoke test on the task list area
3. Observe that no errors or warnings in wp-content/debug.log



### Other information:




### FOR PR REVIEWER ONLY:
